### PR TITLE
[CONSRV] Only call CoUninitialize if initialized

### PR DESCRIPTION
**Pull request by whindsaks**
Only call CoUninitialize on SUCCEEDED

### CODE REVIEW

**Review by SergeGautherie — APPROVED** [review]

File: win32ss/user/winsrv/consrv/console.c [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * PROJECT:         ReactOS Console Server DLL
  * FILE:            win32ss/user/winsrv/consrv/console.c
  * PURPOSE:         Console Management Functions
- * PROGRAMMERS:     G� van Geldorp
+ * PROGRAMMERS:     Gé van Geldorp
```
Comment (JoachimHenze) [inline win32ss/user/winsrv/consrv/console.c:6]: We do have an encoding change here. Is it to the good (towards UTF8 wo BOM)? Or to the bad (away from UTF8woBOM)?

Comment (SergeGautherie) [reply win32ss/user/winsrv/consrv/console.c:6]: Good, according to Hermès's today commit.

Comment (whindsaks) [reply win32ss/user/winsrv/consrv/console.c:6]: Github did that, from 8...something to UTF-8

**Review by HBelusca — APPROVED** [review]
I confirm that the original file was ANSI-encoded.

**Pull request merged**